src/mmw/apps/bigcz/clients/cuahsi.py

```diff
@@ -154,14 +154,14 @@ def get_series_catalog_in_box(box, from_date, to_date):
     to_date = to_date or DATE_MAX
 
     result = make_request(client.service.GetSeriesCatalogForBox2,
-        xmin=box.xmin,
-        xmax=box.xmax,
-        ymin=box.ymin,
-        ymax=box.ymax,
-        conceptKeyword='',
-        networkIDs='',
-        beginDate=from_date.strftime(DATE_FORMAT),
-        endDate=to_date.strftime(DATE_FORMAT))
+                          xmin=box.xmin,
```

Comment:
The actual validation removal seems to have been in the previous commit, and this has only lint changes. Consider rewording the commits to better reflect their contents.